### DISCUSSION TITLE
Table of content and naming pages

### DISCUSSION THREAD

**Discussion by xavierBizoux** [maforget/ComicRackCE#90] [Q&A]
Hi, the question is not 100% related to CR, but may be someone can help. 
I'm trying to add a table of content for my CBR file. I don't know which metadata should be set to display the page name instead of the page number. Do you have an idea?

**Comment by maforget** [comment]
I am so confused about this question. CBR are just images, so how are you trying to create a table of content? 

If your TOC is just an image then metadata would be pointless. Any other way I don't get how you would create a table of content. The only feature related to this are bookmarks in ComicRack. You can add bookmarks if that is what you are asking. But even then where are you using the metadata? An external program reading the `ComicInfo.xml` ? Then that would require to convert to CBZ and not use a CBR.

Then where are you trying to see the Page Name? In the pages view? Then the bookmark is shown in the page view when in Tiles or Detail mode. So what would be the problem? The page name (or bookmark) is already shown there.
